proto-common/src/lib.rs

```diff
@@ -1,3 +1,3 @@
 include!("gen/mod.rs");
 
-pub const SEDA_CHAIN_VERSION: &str = "main";
+pub const SEDA_CHAIN_VERSION: &str = "hy/free-commit-reveal";
```

Comment:
Note to self: This should eventually be updated.

Reply:
This will be updated automatically when we run `cargo proto-update`, but yes, we can update it when that branch is merged, or we we will likely update it once the chain does a release. Especially now that the contract depends on the proto.